berkdb/log/log_put.c

```diff
@@ -1159,14 +1159,11 @@ __log_putr(dblp, lsn, dbt, prev, h)
 	if (lp->num_segments > 1) {
 		tmplsn = *lsn;
 		ret = __log_fill_segments(dblp, lsn, &tmplsn, hdr, nr);
-		assert(tmplsn.offset == lsn->offset + hdr->size);
+		assert(tmplsn.offset == lsn->offset + nr);
 	} else {
 		ret = __log_fill(dblp, lsn, hdr, (u_int32_t)nr);
 	}
 
```

Comment:
Can we really git rid of this?

Reply:
oh - it's because the header isn't used beyond this .. ic

Reply:
Right Mark - `__log_hdrswap` doesn't swap `size` either.